Makefile.am

```diff
@@ -3241,6 +3281,7 @@ intgcheck-prepare:
 	    --without-semanage \
 	    $(INTGCHECK_CONFIGURE_FLAGS); \
 	$(MAKE) $(AM_MAKEFLAGS); \
+	$(MAKE) $(AM_MAKEFLAGS) tcurl-test-tool || echo "libcurl tests will be disabled"; \
```

Comment:
Ahh, you want to use it in integration tests. Then it should not part of test `check_PROGRAMS`
IMHO; there are simpler option add it to `noinst_PROGRAMS` or install it only with integration tests. But we do not have such AM_CONDITION yet